payment/views.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import viewsets, generics, mixins
+from rest_framework.permissions import IsAuthenticated
+
+from payment.models import Payment
+from payment.serializers import PaymentSerializer
+
+
+class PaymentViewSet(
```

Comment:
I guess it makes sense to allow users access to detail views of their payments only.